src/q3/info-FSAB1402/Mooc/Ex mooc.tex

```diff
@@ -0,0 +1,1152 @@
+\documentclass[fr,license=none]{../../../eplsummary}
+\usepackage{wasysym}
+\usepackage{../../../eplcode}
+
+\frenchbsetup{StandardLists=true}
+\renewcommand*{\labelitemi}{\strut\textcolor{black}{$\bullet$}}
+
+\DeclareMathOperator{\pgcd}{PGCD}
+
+\lstset{language={Oz},morekeywords={for,do}}
+
+\hypertitle[']{Informatique : Solutions des exercices du MOOC}{3}{FSAB}{1402}
+{Luis Tascon Gutierrez\and Louis Devillez}
+{Peter Van Roy}
+
+\newcommand{\st}{\mathrm{ST}}
+\newcommand{\ce}{\mathrm{CE}}
+\newcommand{\mozart}{Mozart}
+
+
+\part{Louv1.1x}	
+	\section{Introduction}
+	%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+	%Introduction%
+	%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+	
+		\subsection{Why and how should we learn about paradigms?}
+			\begin{itemize}
+				\item What is a paradigm? \\
+					\textit{''An approach for programming a computer based on a coherent set of principles or a mathematical theory.''}
```

Comment:
Import the package `csquotes` and use the command `\enquote{insert text here}` to avoid mistakes with quote marks.